wiki/Article_Styling_Criteria/en.md

```diff
@@ -1,14 +1,24 @@
 # Article Styling Criteria
 
-_See also: [Article Styling Criteria/News](/wiki/Article_Styling_Criteria/News)._
+*For news posts, see: [News Styling Criteria](/wiki/News_Styling_Criteria)*
 
-The article styling criteria (ASC) serves as a enforced styling standard to keep consistency in clarity, formatting, and layout in all articles of the osu! wiki.
+The Article Styling Criteria (ASC) serves as the osu! wiki's enforced styling standards to keep consistency in clarity, formatting, and layout in all articles.
 
-All articles should try to aim at having proper grammar, correct spelling, and correct information. Keep in mind that reviewers will ask for changes in your pull request(s) for blunders or suggestions. A good osu! wiki writer/editor should read these reviews to help improve the overall quality of these articles to ensure an optimal experience for the reader.
+All articles should try to aim at having proper grammar, correct spelling, and correct information. Remember that reviewers will give you feedback in your pull request(s) for blunders and/or suggestions. A good osu! wiki writer/editor will read these and either apply them or return feedback to help improve the overall quality of the osu! wiki to ensure an optimal experience for the reader.
+
+If you have any suggestions against or regarding the Article Styling Criteria, [open an issue on GitHub](https://github.com/ppy/osu-wiki/issues/new).
 
 ## Repository directory
 
-The following standards apply only to articles saved in the `wiki/` directory.
+The following repository directory standards apply only to files saved in the `wiki` directory.
+
+### Article caching
+
+Articles in the osu! wiki are cached for up to five hours before they are fetched again from GitHub. Purging an article from the cache is possible, but is limited to a few osu! team members and is normally not needed.
```

Comment:
maybe mention Ephemeral like you have in the News article? might not be necessary, your call

Reply:
Hmm, I don't think it's needed

Reply:
Not needed.